DESIGN.md

```diff
@@ -0,0 +1,55 @@
+## Finesse React Project Component Structure
+
+### Root (`src` Folder)
```

Comment:
Would prefer to move the visual (diagram) at the top and the list of folders at the bottom because details and the explanation on how CSS are organized are more "DESIGN" things in my opinion.

DESIGN should also include why and perhaps some overall idea of the UX concept for the frontend.

Reply:
Updated!